src/main/java/hudson/plugins/logparser/LogParserAction.java

```diff
@@ -211,6 +217,8 @@ public String generateToolTip(CategoryDataset dataset, int row,
                     return "Errors: " + result.getTotalErrors();
                 case 1:
                     return "Warnings: " + result.getTotalWarnings();
+                case 2:
+                    return "Debugs: " + result.getTotalDebugs();
```

Comment:
Given that this change is about allowing arbitrary status, I don't think it makes sense to introduce yet another hardcoded `Debugs` status which some users might not want.  In fact, I don't think it makes sense for *any* of these statuses to be hardcoded; they should just be defaults.